docs/machine-learning/tutorials/health-violation-classification-model-builder.md

```diff
@@ -0,0 +1,169 @@
+---
+title: 'Tutorial: Classify health violations with Model Builder'
+description: This tutorial illustrates how to build a multiclass classification model using ML.NET Model Builder to classify restaurant health violation severity in San Francisco.
+author: luisquintanilla
+ms.author: luquinta
+ms.date: 10/25/2019
```

Comment:
```suggestion
ms.date: 10/29/2019
```

Reply:
Good to give yourself the latest date possible to show the article is up to date.